spring-rabbit/src/main/java/org/springframework/amqp/rabbit/connection/RabbitConnectionFactoryBean.java

```diff
@@ -121,6 +123,27 @@
 
 	private volatile SecureRandom secureRandom;
 
+	private boolean skipServerCertificateValidation;
+
+
+	/**
+	 *
+	 * @return whether or not Server Side certificate has to be validated or not
```

Comment:
Would be better to have some general Java Doc description for `public` method. 
The `@return` not enough.
Plus `@since 1.6.6`